### ISSUE TITLE
Should change this division to multiplication? 

### ISSUE THREAD

**Issue by TriLoo** [qfgaohao/pytorch-ssd#52]
https://github.com/qfgaohao/pytorch-ssd/blob/1e86eb3a9655ca1893b537829bbbc2794614d485/vision/utils/box_utils.py#L36

If the centers of prior box are relative to image size, so the `division` to scale should be changed to multiplied by `scale` ???

I have not yet verify this by the training loss function.

**Comment by qfgaohao** [comment]
depends on the value of scale.
